other/rabbitmq/enabled_plugins

```diff
@@ -0,0 +1 @@
+[rabbitmq_management,rabbitmq_consistent_hash_exchange].
```

Comment:
what is this new plugin (hash exchange I mean)?

Reply:
consistent hash ensures that each hijack go to a specific queue and not random to avoid deadlocks